src/sentry/monitors/tasks.py

```diff
@@ -48,7 +49,7 @@ def check_monitors(current_datetime=None):
     )[
         :MONITOR_LIMIT
     ]
-    metrics.gauge("sentry.tasks.check_monitors.missing_count", qs.count())
+    metrics.gauge("sentry.monitors.tasks.check_monitors.missing_count", qs.count())
```

Comment:
will this put the metrics under a new title in datadog?

Reply:
Yes it will. Do we need to maintain the same name?

Reply:
it's probably fine, just gotta change the graphs in the dashboard